pkg/starboard/config.go

```diff
@@ -431,3 +431,23 @@ func (c *configManager) Delete(ctx context.Context) error {
 	}
 	return nil
 }
+
+// DefaultAffinity constructs a new Affinaty resource with default values: linux supported nodes
+func DefaultAffinity() *corev1.Affinity {
```

Comment:
I think the intention is to explicitly schedule to linux nodes. The default is not very meaningful until you read the code of the method.

```suggestion
func LinuxNodeAffinity() *corev1.Affinity {
```